pr_agent/tools/pr_reviewer.py

```diff
@@ -35,6 +35,7 @@ def __init__(self, pr_url: str, cli_mode=False, is_answer: bool = False):
             "description": self.git_provider.get_pr_description(),
             "language": self.main_language,
             "diff": "",  # empty diff for initial calculation
+            "require_score": settings.pr_reviewer.require_score_review,
```

Comment:
Consider initializing the 'require_score' variable with a default value in case the 'require_score_review' setting is not found in the configuration file. This will prevent potential errors if the setting is missing. [important]